integrations/access/common/recipient.go

```diff
@@ -23,6 +23,16 @@ import (
 	"github.com/gravitational/teleport/integrations/lib/stringset"
 )
 
+const (
+	// RecipientKindSchedule shows a recipient is a schedule.
+	RecipientKindSchedule = "schedule"
+)
+
+const (
+	// ReqAnnotationRespondersKey is the request annotation key at which alternate responders are stored.
+	ReqAnnotationRespondersKey = "teleport.dev/responder"
```

Comment:
What is a "responder"? Is it a schedule name basically?

Reply:
swapped to schedules